open-codegen/opengen/icasadi/build.rs

```diff
@@ -20,14 +15,6 @@ fn main() {
         Path::new("extern/auto_casadi_grad.c").exists(),
         "extern/auto_casadi_grad.c is missing"
     );
-    assert!(
-        Path::new("extern/icasadi.c").exists(),
-        "extern/icasadi.c is missing"
-    );
-    assert!(
-        Path::new("extern/icasadi.h").exists(),
-        "extern/icasadi.h is missing"
-    );
     assert!(
         Path::new("extern/icasadi_config.h").exists(),
```

Comment:
Do we use `extern/icasadi_config.h` anywhere?

Reply:
That one should have been removed